roles/confluent.test/molecule/custom-user-plaintext-rhel/verify.yml

```
@@ -0,0 +1,189 @@
+---
+- name: Verify - Zookeeper User Change and custom logging path
+  hosts: zookeeper1-kafka-broker1
+  gather_facts: false
+  tasks:
+    - name: Set Zookeeper user to new user
+      set_fact:
+        zookeeper_user: zk-test-user
+
+    - name: Create Zookeeper User
+      user:
+        name: "{{zookeeper_user}}"
+        comment: "Zookeeper User"
+        system: true
+        group: cp-test-group
+
+    - name: Create Zookeeper Log Directory
+      file:
+        path: /kafka/logs/
+        state: directory
+        recurse: true
+        group: cp-test-group
+        owner: "{{zookeeper_user}}"
+        mode: 0770
+
+    - name: Restart Service
+      systemd:
+        daemon_reload: true
+        name: "{{zookeeper_service_name}}"
+        state: restarted
+
+    - import_role:
+        name: confluent.zookeeper
+        tasks_from: health_check.yml
+
+    - name: Restart Service
+      systemd:
+        daemon_reload: true
+        name: "{{kafka_broker_service_name}}"
+        state: restarted
+
+    - name: Wait for Under Replicated Partitions on Broker
+      include_tasks: ../../../../tasks/wait_for_urp.yml
+    
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: zookeeper-server.log
+        custom_path: /kafka/logs/
```

Why not one var w the full path?

```suggestion
        file_path: /kafka/logs/zookeeper-server.log
```

---

because I want it modular.

---

In what case will two variables help? 

They just get combined together:
```
- name: Check file ownership
  stat:
    path: "{{custom_path}}{{file_name}}"
  register: path_exists
```

So if you just had one "full file path" variable then you can set it to whatever you want.